utils/lib/pb_type_xml.py

```diff
@@ -90,19 +97,180 @@ def add_switchblock_locations(xml):
     })
 
 
-def start_pb_type(
+def start_sub_tile(sub_tile_name, input_wires, output_wires):
```

Comment:
Please provide documentation of expect argument types in some way.  Docstring and/or type annotations are acceptable.

Reply:
Done